apps/laboratory/package.json

```diff
@@ -39,11 +39,9 @@
     "@emotion/styled": "11.11.0",
     "@sentry/browser": "7.92.0",
     "@sentry/react": "7.92.0",
-    "@solana/wallet-adapter-backpack": "0.1.14",
     "@solana/wallet-adapter-wallets": "0.19.32",
     "@solana/web3.js": "1.91.7",
     "@tanstack/react-query": "5.24.8",
-    "@wagmi/core": "2.11.6",
```

Comment:
Are you sure we need to remove this? Lab might need this bc there we are testing wagmi features under wagmi examples @phoenixVS?

Reply:
Moved it back, answered below